lib/taskana-core/src/main/java/pro/taskana/task/internal/ServiceLevelHandler.java

```diff
@@ -66,13 +68,12 @@ public ServiceLevelHandler(
   // - For each task iterate through all referenced classifications and find minimum ServiceLevel
   // - collect the results into a map Duration -> List of tasks
   // - for each duration in this map update due date of all associated tasks
-  public BulkOperationResults<String, TaskanaException> setPlannedPropertyOfTasksImpl(
-      Instant planned, List<String> argTaskIds) {
-    BulkOperationResults<String, TaskanaException> bulkLog = new BulkOperationResults<>();
+  public BulkLog setPlannedPropertyOfTasksImpl(Instant planned, List<String> argTaskIds) {
```

Comment:
Why is this one public while the other operations are not?

Reply:
Good catch. I removed the 'public' modifier